app/components/contact-me.tsx

```diff
@@ -24,7 +24,7 @@ export function ContactMe({ className }: { className?: string }) {
         target="_"
       >
```

Comment:
Added titles to all SVGs, which are applied via SVGR/SVGO. I'm actually not thrilled about this - the plugin seems to always strip out <title> attributes in the files themselves no matter what config I set - but this is a fine fallback for now just to resolve accessibility issues.